stdlib/public/core/DebuggerSupport.swift

```diff
@@ -328,6 +328,26 @@ public enum _DebuggerSupport {
 
     return target
   }
+
+  public static func stringForPrintObject(_ pointer: UnsafeRawPointer?, mangledTypeName: String) -> String {
```

Comment:
I intend to rework this signature, either making it throws, or changing the return type to `Result` or `Optional`.